wpeverest/everest-forms
# Feature - Regex validation for field

### All Submissions:






### Changes proposed in this Pull Request:



Previously, We don't have the option to add custom regular expression validation for the field to validate it. Currently, this option is available for email, first name, last name, input field, number field and website field. The PR will introduce this feature.
### How to test the changes in this Pull Request:

1. Go to form builder then advanced settings of field options.
2. enable regex validation and select a predefined regular expression or input a custom regular expression.
3. update the form and verify whether it is working fine or not.

### Types of changes:




### Other information:




### Changelog entry

> Feature - Regex validation for field.
